connectors/src/connectors/notion/temporal/activities.ts

```diff
@@ -608,47 +599,12 @@ export async function shouldGarbageCollect({
     return false;
   }
 
-  if (
+  // If we are currently doing a full resync, we should not garbage collect
+  const isDoingAFullResync =
```

Comment:
nit: this is a slight misnomer as we could also be doing the intial sync (which isn't a "re" sync), correct ?

Reply:
Maybe we should rename this entire activity into "isFullSyncOngoing" ? or something (not strongly held though)

Reply:
> Maybe we should rename this entire activity into "isFullSyncOngoing" ? or something (not strongly held though)

Yes, now that it's only called by garbage collection workflow, it makes total sense.